js/waiting.js

```diff
@@ -0,0 +1,34 @@
+(function() {
+  var s = (new Date()).getTime();
+  var tmp = new moment.duration(moment(new Date(launchDate).getTime())-s);
+  var countdownStr = function(tms) {
+    return tms.days() + 'd ' + tms.hours() + 'h ' + tms.minutes() + 'm ' + (tms.seconds() <= 9 ? '0' + tms.seconds() : tms.seconds()) + 's';
+  };
+  $('#countdown').html(countdownStr(tmp));
+  $.ajax({
+    url: 'time.php',
+    data: {
+      t: (new Date()).getTime()
+    },
+    success: function(resp) {
```

Comment:
I think this should have an error handler as well though I don't have immediate ideas for good content to show the user.

Reply:
addressed in commit: c1b63e5b775889446974e4509b87fa5a2dd2b7cb

sorry, lumped some things together in this commit